nixos/modules/services/misc/forgejo.nix

```diff
@@ -621,14 +627,10 @@ in
       environment = {
         USER = cfg.user;
         HOME = cfg.stateDir;
-        # `GITEA_` prefix until https://codeberg.org/forgejo/forgejo/issues/497
-        # is resolved.
-        GITEA_WORK_DIR = cfg.stateDir;
-        GITEA_CUSTOM = cfg.customDir;
       };
     };
 
-    services.openssh.settings.AcceptEnv = mkIf (!cfg.settings.START_SSH_SERVER or false) "GIT_PROTOCOL";
+    services.openssh.settings.AcceptEnv = mkIf (!cfg.settings.server.START_SSH_SERVER or false) "GIT_PROTOCOL";
```

Comment:
I think I would prefer it, if you could more this into another commit.
So two commits in total.

One for the `GITEA_`/`FORGEJO_` prefix thingy, and one for fixing #306205.

Reply:
These have been split out